docs/docs-content/clusters/cluster-management/platform-settings/pause-platform-upgrades.md

```diff
@@ -8,11 +8,58 @@ tags: ["clusters", "cluster management"]
 ---
 
 Palette supports the **Pause Agent Upgrades** feature to exclude a cluster or a group of clusters from getting
-automatically upgraded when Palette is upgraded. The three ways to activate this feature are:
+automatically upgraded when Palette is upgraded.
 
-- Pause Upgrades for a Single Cluster
-- Pause Upgrades for all Clusters within Project Scope
-- Pause Upgrades for all Clusters within Tenant Scope
+## Pause Agent Upgrade Scopes
+
+Upgrades can be paused and resumed in the following scopes:
+
+- Pause upgrades for a single cluster
+- Pause upgrades for all clusters within a project
+- Pause upgrades for all clusters within a tenant
+
+When determining if the agent upgrades for one cluster is paused or not, you only need to look at the setting for the
```

Comment:
Maybe:
"Agent upgrade settings are **always** applied based on individual cluster settings. Project and Tenant agent upgrade settings are not inherited - instead cluster level settings are set to match each time Project and Tenant level settings are changed."